cmd/command/recovery.go

```diff
@@ -85,22 +89,46 @@ func recoverObjectAction(ctx *cli.Context) error {
 	}
 
 	if ctx.IsSet(objectListFlag.Name) {
-		if ctx.NArg() < 1 {
-			return fmt.Errorf("args number error")
-		}
-		argNum := ctx.NArg()
-		for i := 0; i < argNum-1; i++ {
-			recoverObject(bucketName, ctx.Args().Get(i), cfg, client)
+		objectNames := ctx.String(objectListFlag.Name)
+		objectNameList := strings.Split(objectNames, "//")
+		for _, objectName := range objectNameList {
+			recoverObject(bucketName, objectName, cfg, client)
 		}
 	} else {
 		objectName := ctx.String(objectFlag.Name)
 		recoverObject(bucketName, objectName, cfg, client)
 	}
+
+	taskCheckTicker := time.NewTicker(RecoveryJobSyncTime)
+
+	for range taskCheckTicker.C {
+		stats, _ := client.GetTasksStats(context.Background())
+		if stats.GetRecoveryProcessCount() > 0 {
+			log.Infof("Processing recovery tasks, update every 5 seconds. Waiting Tasks Number: %d. Recovered: %s. Failed: %s", stats.GetRecoveryProcessCount(), stats.GetRecoverySucceedList(), stats.GetRecoveryFailedList())
```

Comment:
If it is information for the user, it is recommended not to use log printing, but to directly use fmt.Println to print in separate lines. three lines for the three stat value

Reply:
Got it, updated to fmt.Printf and use \n to separate lines. The output format is currently as below:
Processing recovery tasks, update every 5 seconds.
Waiting Tasks Number: 1
Recovered: [objectName1 objectName2]
Failed: []